scripts/selinux/README.md

```diff
@@ -36,16 +44,42 @@ sudo restorecon -R /etc/nginx-agent
 ```
 
 ## Updating existing policy
-Copy the folder `scripts/selinux` over to your rhel server.
+Copy the folder `scripts/selinux` over to your rhel 8 server.
```

Comment:
would it be worth mentioning that you need to change the file permission on nginx_agent.sh ?

Reply:
Good point. I'll add that